.github/actions/set-aproxy-bypass-for-testflinger/action.yml

```diff
@@ -7,10 +7,9 @@ runs:
     - name: Reconfigure aproxy/nftables
       shell: bash
       run: |
-        sudo apt install dnsutils
         sudo snap set aproxy listen=:28443
         sudo nft -f - << EOF
-        define testflinger-ips = { $(dig +short testflinger.canonical.com | grep -o '[0-9]\{1,3\}\.[0-9]\{1,3\}\.[0-9]\{1,3\}\.[0-9]\{1,3\}' | tr '\n' ',') }
+        define testflinger-ips = { $(getent ahostsv4 testflinger.canonical.com | awk '/RAW/ {print $1}' | paste -sd,) }
```

Comment:
haha, cool stuff! Thanks, Simon!

Reply:
Thanks for teaching me about `\K` in `perl`, I wasn't aware of this ;)